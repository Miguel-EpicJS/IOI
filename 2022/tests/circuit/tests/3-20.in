385045b28fbc1903eca997d513040235e0229e01
510 819 5
-1 0 0 1 3 3 2 1 4 0 0 5 0 0 13 2 6 15 11 3 4 2 0 6 2 13 15 22 15 23 10 21 3 24 18 17 17 24 4 3 29 40 16 11 10 28 33 38 39 10 38 28 51 41 50 4 3 56 7 44 44 13 9 49 51 23 39 42 23 40 60 1 63 62 55 62 70 55 64 58 50 34 23 80 71 14 71 56 48 79 8 38 28 92 62 61 73 8 46 16 93 56 100 67 23 92 104 99 103 77 105 93 86 91 54 99 21 29 59 94 105 94 3 86 51 119 106 32 3 93 111 65 8 125 51 130 133 122 100 129 70 106 9 138 112 115 116 87 86 91 61 120 75 138 142 122 145 138 92 142 142 125 110 158 146 94 72 134 154 158 119 129 74 159 159 109 131 131 171 100 179 33 108 180 50 177 17 126 168 186 129 190 176 103 159 104 174 65 128 168 160 127 147 198 197 175 171 49 160 179 198 174 200 192 197 150 194 35 203 115 195 151 184 216 214 194 200 111 129 99 101 223 216 231 186 188 206 235 223 174 180 238 170 168 227 237 205 211 95 220 170 92 251 33 173 203 171 214 239 230 244 241 205 160 244 217 128 263 148 201 194 227 256 154 159 235 226 169 220 265 275 219 210 281 212 276 240 195 285 272 229 276 285 292 202 219 255 190 273 274 160 260 252 298 239 267 154 288 290 290 273 305 290 288 270 300 217 312 311 232 296 267 198 316 263 266 251 309 176 292 304 296 266 322 310 240 212 330 232 307 330 308 123 286 341 321 287 299 332 347 298 341 322 289 304 338 345 345 226 321 274 346 340 360 225 357 252 316 365 364 353 134 347 328 363 371 348 360 367 377 267 291 132 372 336 310 337 359 364 220 359 379 250 386 330 375 193 324 350 334 399 391 321 321 402 217 309 357 373 382 380 336 401 371 382 397 341 415 300 350 387 306 414 344 397 423 402 424 427 399 408 420 426 417 403 415 434 300 432 418 417 432 405 428 430 437 303 408 436 411 445 442 376 420 402 439 397 438 425 379 454 451 450 455 461 456 441 437 436 448 464 454 466 471 458 447 468 459 452 462 411 387 437 480 480 459 477 484 473 473 470 484 474 491 460 494 468 483 454 494 499 439 498 461 488 504 504 506 500 508 0 1 3 7 7 8 8 18 19 20 12 25 20 22 27 12 12 26 32 14 15 27 16 31 22 47 43 30 16 49 49 32 43 15 13 46 17 38 37 30 6 28 59 61 35 22 44 31 25 60 5 55 47 55 50 55 9 34 69 21 50 58 74 67 41 62 22 9 82 70 73 41 61 75 23 85 84 77 23 81 80 83 45 78 69 85 42 88 44 57 92 76 63 48 93 35 27 85 82 103 66 78 89 37 96 95 40 88 83 86 87 84 53 100 107 115 118 76 109 42 111 15 89 72 102 87 23 87 114 36 26 12 114 118 123 14 54 52 124 41 97 107 64 122 40 112 115 1 86 90 0 122 116 100 137 130 136 45 144 107 143 57 127 68 101 121 98 84 157 68 102 127 126 30 132 80 127 147 146 139 121 139 138 100 53 148 98 52 134 127 114 121 76 152 123 138 163 160 162 157 160 143 138 161 141 97 156 119 166 141 129 113 175 106 50 165 140 166 171 177 156 135 168 90 163 135 167 117 149 172 187 183 187 110 117 190 187 162 182 184 192 144 158 113 196 189 155 195 176 194 161 167 196 170 124 184 149 198 165 184 203 175 185 215 199 155 79 36 60 208 220 206 133 55 198 207 202 153 181 221 164 211 171 224 192 181 66 170 113 178 152 209 162 230 215 195 164 200 96 216 191 137 225 241 204 193 213 218 236 126 185 231 216 174 19 101 132 248 232 11 229 182 248 225 200 246 233 153 245 202 166 191 151 246 189 139 253 253 240 231 244 242 238 247 169 224 242 242 250 233 184 264 221 224 258 140 251 249 258 254 120 136 207 265 148 270 234 271 252 127 259 277 251 237 222 145 254 278 211 279 171 280 282 234 128 260 150 273 213 183 287 255 256 187 271 289 291 172 219 279 275 293 281 108 284 295 273 178 295 294 243 300 209 285 283 287 269 247 210 272 297 218 308 280 262 276 285 81 166 257 300 283 311 263 191 220 262 201 264 318 304 307 306 293 266 319 222 325 250 268 319 228 282 324 330 314 327 249 300 302 204 245 331 278 325 236 301 188 313 268 326 333 302 286 340 321 332 243 317 297 339 257 277 327 344 259 323 300 301 299 235 318 356 313 339 356 346 261 352 351 333 115 303 347 360 335 325 351 329 269 323 339 355 309 358 363 366 329 315 346 346 355 361 361 348 328 241 369 354 342 208 384 380 284 385 372 345 389 173 365 320 294 335 314 373 390 367 326 380 263 199 263 383 397 398 373 313 312 384 343 396 315 384 395 382 371 404 228 378 337 385 345 393 338 378 342 320 385 374 354 344 379 405 352 317 396 405 381 408 339 342 394 310 368 372 370 369 394 376 362 329 412 409 397 398 353 406 400 381 410 321 376 382 409 368 388 393 391 349 406 387 422 383 300 390 388 414 413 422 424 425 327 420 366 370 416 419 386 429 395 261 396 334 400 401 424 418 432 374 368 426 403 392 435 337 433 419 358 410 403 440 375 413 407 439 361 428 440 372 437 442 421 421 438 448 389 431 441 430 429 446 442 445 442 423 444 450 425 362 412 452 435 343 443 453 413 457 305 453 443 435 455 427 446 377 451 453 463 392 440 456 462 455 250 427 415 449 430 404 444 463 449 416 463 458 433 331 467 438 281 418 469 434 470 477 465 455 407 472 435 460 464 475 469 476 479 457 471 412 483 447 458 474 349 485 481 431 468 482 472 478 455 482 486 476 480 483 488 492 487 486 495 465 466 485 499 475 493 489 497 479 501 493 489 461 491 500 492 501 457 496 498 481 496 467 490 478 499 503 503 490 500 505 502 495 487 502 476 497 507 506 505 507 508 503 504 508 509 509
1 0 1 0 0 1 0 0 1 0 1 0 1 0 1 0 1 1 0 0 1 0 0 1 0 1 0 0 0 1 1 0 0 1 1 0 0 0 1 1 0 0 0 0 0 1 1 1 0 0 0 1 0 1 1 1 0 0 0 1 1 0 1 1 0 0 0 1 0 1 0 1 0 1 0 0 0 0 0 0 0 1 0 0 0 1 0 1 0 0 0 1 0 1 1 1 1 0 0 0 0 0 1 0 1 0 0 0 0 0 1 1 0 1 0 0 0 1 0 1 1 1 1 0 0 0 0 0 0 0 0 0 1 1 1 1 0 1 0 0 1 0 1 1 0 1 1 1 0 1 0 0 1 1 1 0 1 1 0 0 0 1 1 0 0 0 1 0 1 0 1 1 0 0 1 0 1 0 1 0 1 1 1 0 0 0 0 1 0 1 1 0 1 1 0 1 1 1 1 1 1 1 1 1 0 1 0 0 0 0 1 1 0 0 0 0 1 1 0 1 1 1 1 0 0 0 0 1 1 0 0 0 1 0 1 1 0 1 1 1 1 0 0 0 0 1 1 0 1 1 0 0 1 0 0 1 1 0 0 1 0 1 0 0 0 0 0 0 0 1 1 1 0 0 1 0 0 1 1 0 0 1 0 1 0 0 0 1 1 0 0 1 1 0 0 0 0 0 0 0 1 1 0 0 0 0 0 1 0 1 0 0 1 1 0 0 0 1 0 1 1 0 1 1 0 1 0 0 1 1 1 1 1 0 0 0 0 1 1 1 0 0 1 0 0 0 0 1 1 0 1 0 0 0 0 1 0 0 1 1 1 0 0 1 0 0 0 0 1 1 1 1 0 0 1 0 1 1 1 1 0 1 1 0 1 0 0 0 0 0 1 1 0 0 1 1 0 0 0 1 1 0 0 0 0 1 1 0 0 1 0 0 1 0 1 0 1 0 0 1 1 1 0 1 0 0 1 1 0 0 1 0 1 0 1 0 0 1 0 0 0 1 1 1 0 0 1 1 1 1 1 1 0 0 0 1 0 1 1 1 1 1 1 1 1 1 0 0 1 1 1 0 1 0 1 0 1 0 1 1 0 1 0 1 0 0 0 1 1 1 0 1 0 1 1 0 0 0 0 1 0 1 1 1 0 0 1 1 1 1 0 0 1 0 1 1 0 1 0 0 1 0 1 1 0 1 0 1 0 0 1 0 0 0 1 1 0 1 0 0 1 0 0 0 0 0 1 1 1 1 0 1 1 0 0 1 0 1 1 1 0 1 0 0 1 1 0 0 0 0 0 1 0 1 0 0 1 1 1 0 1 0 0 0 1 1 1 1 1 1 1 1 0 0 0 1 0 0 0 1 0 0 0 0 1 1 1 1 0 0 0 1 0 0 0 0 0 1 0 1 0 0 0 0 1 1 0 1 1 0 1 1 0 1 0 1 0 0 0 0 0 1 1 1 0 1 0 1 1 0 0 0 0 1 1 0 1 0 1 1 1 1 0 0 1 1 0 1 0 0 1 1 1 1 1 1 0 1 0 1 0 0 1 1 0 0 0 0 0 1 1 0 0 1 1 1 1 1 0 0 0 0 0 1 0 0 1 0 0 0 0 0 0 0 1 0 1 1 0 1 0 1 1 1 0 1 1 0 0 1 1 1 0 0 0 1 0 0 1 1 1 0 0 0 0 0 0 1 1 1 1 1 0 0 0 0 1 1 1 0 0 0 1 1 1 1 0 0 0 1 0 0 0 1 1 0 0 1 1 0 0 1 0 1 0 1 0 1 1 0 1 1 0 0 0 1 0 0 0 1 0 0 0 1 1 0 1 0 0 0 1 1 1 1 1 1 1 1 1
1197 1321
832 1082
728 770
793 1005
746 961
