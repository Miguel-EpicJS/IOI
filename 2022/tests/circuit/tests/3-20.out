4faa78c22ee35083e49b65601698a1bd98a87a0f
OK
216372980
926486360
572278876
442949060
582680282
